pkg/app/piped/platformprovider/ecs/client.go

```diff
@@ -150,6 +150,32 @@ func (c *client) RegisterTaskDefinition(ctx context.Context, taskDefinition type
 	return output.TaskDefinition, nil
 }
 
+func (c *client) RunTask(
+	ctx context.Context,
+	taskDefinition types.TaskDefinition,
+	capacityProviderStrategy []types.CapacityProviderStrategyItem,
+	clusterArn string,
+	launchType string,
+	awsVpcConfiguration *types.AwsVpcConfiguration,
+) ([]types.Task, []types.Failure, error) {
```

Comment:
Why don't we just return the error? The outputs aren't used anyway, isn't it?

Reply:
I think you are right. I'll fix it!